paddlets/analysis/analysis_report.py

```diff
@@ -233,7 +233,7 @@ def _get_analyzers(self, names: Union[str, List[str]], params: Dict = None) -> L
 
         return analyzers
 
-    def get_all_analyzers_names(self, log: bool = True) -> List[str]:
+    def get_all_analyzers_names(self, log: Optional[bool] = True) -> List[str]:
```

Comment:
should Not be Optional, refers to https://docs.python.org/3/library/typing.html#typing.Optional